src/code_80057C60.c

```diff
@@ -6798,37 +6838,37 @@ void func_8006E058(void) {
             break;
         case SCREEN_MODE_3P_4P_SPLITSCREEN:
             if ((VERSUS == gModeSelection) || (BATTLE == gModeSelection)) {
-                func_8006E420(gPlayerOne, 0, 0);
-                func_8006E420(gPlayerTwo, 1, 0);
-                func_8006E420(gPlayerThree, 2, 0);
+                func_8006E420(gPlayerOne, PLAYER_ONE, 0);
+                func_8006E420(gPlayerTwo, PLAYER_TWO, 0);
+                func_8006E420(gPlayerThree, PLAYER_THREE, 0);
 
                 if (gPlayerCountSelection1 == 4) {
-                    func_8006E420(gPlayerFour, 3, 0);
+                    func_8006E420(gPlayerFour, PLAYER_FOUR, 0);
                 }
             }
 
             break;
     }
 }
 
-void func_8006E420(Player* player, s8 arg1, s8 arg2) {
+void func_8006E420(Player* player, s8 playerId, s8 arg2) {
     s16 temp_s0;
 
     if ((player->type & PLAYER_EXISTS) == PLAYER_EXISTS) {
         if ((player->type & PLAYER_HUMAN) == PLAYER_HUMAN) {
-            func_8006D194(player, arg1, arg2);
+            func_8006D194(player, playerId, arg2);
         }
 
         for (temp_s0 = 0; temp_s0 < 10; ++temp_s0) {
-            func_8006CEC0(player, temp_s0, arg1, arg2);
+            func_8006CEC0(player, temp_s0, playerId, arg2);
             if (((player->type & PLAYER_HUMAN) == PLAYER_HUMAN) || (gGamestate == ENDING)) {
-                func_8006C9B8(player, temp_s0, arg1, arg2);
+                func_8006C9B8(player, temp_s0, playerId, arg2);
             }
-            func_8006C6AC(player, temp_s0, arg1, arg2);
+            func_8006C6AC(player, temp_s0, playerId, arg2);
         }
 
         if (gModeSelection == BATTLE) {
-            func_8006B9CC(player, arg1);
+            update_player_balloons_position(player, playerId);
```

Comment:
This could probably be attach.

player_attach_balloon_component ?

Reply:
hmm don't really feel like attach or a component